### PR TITLE
Update verify bam id docker pd 2291

### PR DESCRIPTION
**Pull request by nikellepetrillo**
### Description

This docker was failing trivvy scans. It been updated and are passing trivvy scans.

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

**Comment by nikellepetrillo** [comment]
sci exome germline ss test running: https://gotc-jenkins.dsp-techops.broadinstitute.org/job/warp-workflow-tests/51798/

**Review by kevinpalis — APPROVED** [review]
LGTM!

**Review by jessicaway — APPROVED** [review]
Thanks @nikellepetrillo!

File: tasks/broad/BamProcessing.wdl [hunk 1]
```diff
@@ -464,7 +468,9 @@ task CheckContamination {
     preemptible: preemptible_tries
     memory: "7.5 GiB"
     disks: "local-disk " + disk_size + " HDD"
-    docker: "us.gcr.io/broad-gotc-prod/verify-bam-id:1.0.1-c1cba76e979904eb69c31520a0d7f5be63c72253-1639071840"
+    # this docker image was built using the commit hash c1cba76e979904eb69c31520a0d7f5be63c72253 from the VerifyBamID repo
```
[the remark this answers is not in the document]
Comment (nikellepetrillo) [reply tasks/broad/BamProcessing.wdl:471]: Oops, I totally missed this comment before I merged 😬

[the remark this answers is not in the document]
Comment (jessicaway) [reply tasks/broad/BamProcessing.wdl:471]: No worries!

**Pull request merged**